### ISSUE TITLE
Meteor 1.8 and @babel >= 7.0.0

### ISSUE THREAD

**Issue by jmdsg** [nathantreid/meteor-static-assets#3]
Running with meteor 1.8 throws an error like:

>    While loading plugin `static-assets` from package `nathantreid:static-assets`:
   module.js:547:15: Cannot find module
   'C:\Users\Lilian\AppData\Local\.meteor\packages\meteor-tool\1.8.0\mt-os.windows.x86_64\dev_bundle\lib\node_modules\@babel\runtime\helpers\builtin\extends'
   at Function.Module._resolveFilename (module.js:547:15)
   at Function.Module._load (module.js:474:25)
   at Module.require (module.js:596:17)
   at require (internal/module.js:11:18)
   at Object.require (C:\tools\isobuild\bundler.js:2083:22)
   at makeInstallerOptions.fallback (packages/modules-runtime.js:651:18)
   at require (packages/modules-runtime.js:244:16)
   at options.js (<runJavaScript-31>:47:16)
   at fileEvaluate (packages/modules-runtime.js:343:9)
   at require (packages/modules-runtime.js:238:16)
   at <runJavaScript-31>:320:1
   at <runJavaScript-31>:327:3

Updating from:

```javascript
  use: [
    'caching-compiler@1.1.9',
    'ecmascript@0.9.0',
  ],
```

to

```javascript
  use: [
    'caching-compiler@1.2.0',
    'ecmascript@0.12.0',
  ],
```

solves the issue.

**Comment by nathantreid** [comment]
Thanks! I'll get this fixed soon.

**Comment by nathantreid** [comment]
Just an update - I've been extremely busy of late and haven't been able to get to this yet. It's still on my radar!

**Comment by nathantreid** [comment]
This is resolved in `nathantreid:static-assets@0.1.0`.
